Library/Homebrew/extend/os/linux/development_tools.rb

```diff
@@ -19,5 +19,9 @@ def locate(tool)
     def default_compiler
       :gcc
     end
+
+    def build_system_info
+      generic_build_system_info.merge "glibc_version" => OS::Linux::Glibc.system_version
```

Comment:
Since we want

> the Glibc version that was used to build the bottle on Linux

my suggestion would be:

```rb
(Formula["glibc"].any_installed_version&.version || OS::Linux::Glibc.system_version)
```

if the `glibc` formula is guaranteed to be prioritised over system `glibc` when installed.

(I know it doesn't matter for our CI, but I think it's good to make sure it is correct anyway)

Reply:
> The only catch here however, is perhaps if we are writing a tab for glibc itself. Might end up picking itself.

That's fine. The host `glibc` version doesn't affect the brewed `glibc`.

Reply:
Done!